tests/vmi_configuration_test.go

```diff
@@ -2212,18 +2212,24 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 		})
 
 		It("[test_id:5360]should set appropriate IO modes", func() {
-			vmi := libvmifact.NewCirros(
+			libstorage.CreateHostPathPv("alpine-host-path", testsuite.GetTestNamespace(nil), testsuite.HostPathAlpine)
+			libstorage.CreateHostPathPVC("alpine-host-path", testsuite.GetTestNamespace(nil), "1Gi")
+			vmi := libvmi.New(
 				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
 				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithResourceMemory("128Mi"),
```

Comment:
This can be an issue depending on the used arch, ARM64 needs `256Mi`.

Can't you keep `libvmifact.NewCirros` instead? I think it is okay for the disk to have name `disk0` instead of `ephemeral-disk0`. If that is the only issue.

Reply:
No, the issue is the added cloud-init volume, and 256Mi is not a problem in this test since we don't need to actually boot

Reply:
Can you just ignore the cloud-init volume?

Reply:
If the test is not supposed to interact with the guest and you don't care about the guest successfully booting, what about omitting the options we don't need? Why set memory or interfaces if you don't care about them?

Reply:
Dropped the interfaces :+1: